File: packages/lexigraph-core/tests/corpus/typescript/nested.ts
type Deep = {
  a: { b: { c: { d: { e: { f: { g: string } } } } } };
  list: Array<Array<Array<Array<number>>>>;
};

const x: Deep = {
  a: { b: { c: { d: { e: { f: { g: 'leaf' } } } } } },
  list: [[[[1, 2, 3]]]],
};

export default x;

File: packages/lexigraph-core/tests/corpus/python/nested.py
def deep():
    if True:
        for i in range(3):
            while i > 0:
                try:
                    if i % 2 == 0:
                        return [[[[[[i]]]]]]
                    else:
                        i -= 1
                except Exception:
                    pass
                finally:
                    pass
    return None
